test/graphql/types/Venue/creator.test.ts

```
@@ -0,0 +1,379 @@
+	currentClient: CurrentClient;
+	drizzleClient: {
+		query: {
+			usersTable: {
+				findFirst: ReturnType<typeof vi.fn>;
+			};
+		};
+	};
+	log: {
+		error: ReturnType<typeof vi.fn>;
+	};
+	app: any; // Add required MercuriusContext properties
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)